### ISSUE TITLE
Helm version [3.14.0, 3.14.2] is broken for Artifactory users using alias

### ISSUE THREAD

**Issue by mikaello** [mend/renovate-ce-ee#493]
In v7.0.0 full image you upgraded Helm to version 3.14.2, this version is broken for Artifactory users using alias config. Could you please upgrade to 3.14.3 or later?

See https://github.com/helm/helm/issues/12748 for more information.

**Comment by rarkins** [comment]
Hi @mikaello we'll prioritize putting out a patch release this week.

Do you run the full image with binarySource=global (i.e. no dynamic runtime installs)?

Are you able to do a custom build in the meantime if you need? (e.g. FROM ..... RUN .....) because we can give instructions for that if it's urgent.

**Comment by mikaello** [comment]
This issue is fixed in v7.1.0:

```
$ docker run ghcr.io/mend/renovate-ee-worker:7.1.0-full helm version
version.BuildInfo{Version:"v3.14.4", GitCommit:"81c902a123462fd4052bc5e9aa9c513c4c8fc142", GitTreeState:"clean", GoVersion:"go1.21.9"}
```
